src/redisjson.rs

```diff
@@ -505,27 +519,35 @@ pub mod type_methods {
         raw::save_string(rdb, &json.data.to_string());
     }
 
-    #[allow(non_snake_case, unused)]
     pub unsafe extern "C" fn aux_load(rdb: *mut raw::RedisModuleIO, encver: i32, when: i32) -> i32 {
```

Comment:
Can't we drop the aux load? it was never released in any official version no?
@gkorland ^^^

Reply:
If we can not drop it then we can only support a short read from version 6.2.5 and we need to check for it.

Reply:
Looking at issue #318 (this [comment](https://github.com/RedisJSON/RedisJSON/issues/318#issuecomment-824620167)), it seems like we need `aux_load`

Reply:
After discussion with @gkorland and @MeirShpilraien, It was decided to drop `aux_load` (no need to support unofficial releases, users had plenty of time to migrate)